doc/source/administrator/security.md

```diff
@@ -73,8 +77,26 @@ If you have your own HTTPS certificates & want to use those instead of the autom
             -----END CERTIFICATE-----
     ```
 
-2.  Apply the config changes by running helm upgrade ....
-3.  Wait for about a minute, now your hub should be HTTPS enabled!
+2. Apply the config changes by running helm upgrade ....
+3. Wait for about a minute, now your hub should be HTTPS enabled!
+
+
+#### Specify certificate through Secret resource
+
+1. Create a `secret` resource with type `kubernetes.io/tls` containing your certificate. Add your domain and the name of your `secret` to your config.yaml.
```

Comment:
Make this two points, where the initial one could be a `kubectl create secret` command or similar.

Reply:
The reason I became aware of this option was because I wanted to load the certificate from an azure key vault. My secret was created by [azure vault to kubernetes](https://github.com/SparebankenVest/azure-key-vault-to-kubernetes). I agree adding the command would be helpful but do you think it's worth including something about cloud service integration (with azure in this case) or do you think that is beyond the scope of this documentation?

Reply:
I have added an example of how to create a secret resource